docs/dev-guide/conventions/sync-v-async.rst

```diff
@@ -15,10 +15,16 @@ Any REST API will return one of three responses:
 A success response indicates no conflicts were detected and the REST call
 executed. This is what is typically expected from a REST API call.
 
-A postponed response indicates that some portion of the command has been
-queued to execute asynchronously.  In this case a :ref:`call_report` will be returned
-with the results of the synchronously executed portion of the command, if there are any,
-and a list of the tasks that have been spawned to complete the work in the future.
+A postponed response indicates that some portion of the command has been queued to execute
+asynchronously.  In this case a :ref:`call_report` or a :ref:`group_call_report` will be returned
+with the results of the synchronously executed portion of the command.
+
+When a single task is queued or a task with follow up tasks, such as a publish after a sync or unbind
```

Comment:
consider removing `or unbind consumers after repo delete`. Two examples aren't that much better than 1 and they make it more confusing.